lib/ci.js

```diff
@@ -22,7 +23,8 @@ const CI_TYPES = new Map([
   [LIBUV, { name: 'libuv', re: /libuv/ }],
   [NOINTL, { name: 'No Intl', re: /nointl/ }],
   [V8, { name: 'V8', re: /node-test-commit-v8/ }],
-  [LINTER, { name: 'Linter', re: /node-test-linter/ }]
+  [LINTER, { name: 'Linter', re: /node-test-linter/ }],
+  [LIGHT, { name: 'Lite Commit', re: /node-test-commit-light/ }]
```

Comment:
There is also node-test-pull-request-lite which is the parent job of node-test-commit-light

Reply:
That's currently caught by the regexp for `FULL` above.

Reply:
I will add that in too then, funny enough my PR title has this in it since i was confused between lite and light.

@richardlau is not caught by `FULL` regex since is closed at the end by a slash (`\/`)

Reply:
The trailing slash is only on the right side of `|`. The left side would match.

Reply:
Yep, its being detected i can confirm (using devtools):
```javascript
> /node-test-pull-request|node-test-commit\//.test('https://ci.nodejs.org/node-test-pull-request-lite/123')
true
```

Reply:
FYI @gibfahn has just renamed the `node-test-commit-lite` job: https://github.com/nodejs/node-core-utils/issues/214#issue-303916863

Reply:
I will update the PR accordingly. Thanks, @richardlau for the heads up